packages/common/src/templates/icons.ts

```diff
@@ -24,6 +24,7 @@ import {
   SapperIcon,
   ParcelIcon,
   VuePressIcon,
+  StackbitIcon,
```

Comment:
Swap out for the Unibit Icon

```suggestion
  UnibitIcon,
```